### DISCUSSION TITLE
Changing git repository with one of which I am not the owner but a collaborator

### DISCUSSION THREAD

**Discussion by marcoruffelli** [vercel/community#6093] [Help]
### Summary

HI,
I have a problem changing the github repository of which I am not the owner, but only shared with me. 
I managed to disconnect the old repository from Project->setting->Git, but when I try to connect another repository it doesn't appear in the list, where there are only those I own. And the "Import Third-Party Git Repository" link does not appear. 
Do I have to delete the project with the verified environment variables and domains and create a new one? 
Or is there some setting I don't see?

Kind Regards

### Steps to Reproduce

Disconnect repository from Project->settings->Git
Connect a repository of which you are not the owner but only collaborators. Example: externaldomain/myproject

**Comment by versecafe** [comment]
@marcoruffelli Are you on the Hobby plan? if so you cannot use a repo you do not own, but you can host a fork of the project, or upgrade to team.
